backends/tfhe-cuda-backend/cuda/include/programmable_bootstrap_multibit.h

```diff
@@ -19,15 +19,16 @@ void scratch_cuda_multi_bit_programmable_bootstrap_64(
     void *stream, uint32_t gpu_index, int8_t **pbs_buffer,
     uint32_t lwe_dimension, uint32_t glwe_dimension, uint32_t polynomial_size,
     uint32_t level_count, uint32_t grouping_factor,
-    uint32_t input_lwe_ciphertext_count, bool allocate_gpu_memory);
+    uint32_t input_lwe_ciphertext_count, bool allocate_gpu_memory,
+    uint32_t chunk_size = 0);
```

Comment:
The lwe_chunk_size shouldn't be there anymore

Reply:
But there should be some change in the scratch arguments for classic & programmable bootstrap to add the pbs_type thing no?

Reply:
I was initially trying to implement an extra argument to skip MS on PBS and that made rebasing a bit more confusing. Clearly this was a fragment of that time. I will fix this.